Scroller/Scroller.js

```diff
@@ -112,19 +377,24 @@ Scroller.propTypes = /** @lends agate/Scroller.Scroller.prototype */ {
 };
 
 Scroller.defaultProps = {
-	'data-spotlight-container-disabled': false, // eslint-disable-line react/default-props-match-prop-types
+	'data-spotlight-container-disabled': false,
+	cbScrollTo: nop,
 	direction: 'both',
-	focusableScrollbar: false, // eslint-disable-line react/default-props-match-prop-types
+	focusableScrollbar: false,
 	horizontalScrollbar: 'auto',
```

Comment:
Do we need a `noScrollByDrag: false` here?

Reply:
Actually we are setting `noScrollByDrag: noScrollByDrag: !platform.touchscreen,` in `useScroll` so I'm going to remove this prop here.